controllers/binding/snapshotenvironmentbinding_adapter_test.go

```diff
@@ -588,6 +552,54 @@ var _ = Describe("Binding Adapter", Ordered, func() {
 			Expect(ok).To(BeTrue())
 			Expect(detail.Status).To(Equal(gitops.IntegrationTestStatusDeploymentError))
 		})
+	})
 
+	It("ensures ephemeral environment is deleted for the given pipelineRun", func() {
```

Comment:
I think we don't need to move this case here as long as as case `Requeues the environment cleanup if the binding is younger than the threshold` use the mocked dtc.

Reply:
moved back.